src/components/UserInfo.js

```diff
@@ -1,20 +1,20 @@
 export default class UserInfo {
-  constructor(name, description) {
+  constructor(name, about) {
     this._name = name;
-    this._description = description;
+    this._about = about;
   }
 
   getUserInfo() {
     //returns the object with info about the user
     const userData = {};
     userData["profileName"] = this._name.textContent;
-    userData["profileDescription"] = this._description.textContent;
+    userData["profileAbout"] = this._about.textContent;
     return userData;
   }
```

Comment:
You should also add the method for updating user avatar `setAvatar` and use it when you need to update the avatar